.github/workflows/generate-builds.yml

```diff
@@ -221,6 +221,9 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y ninja-build
+        sudo apt-get remove -y cmake
+        wget https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-aarch64.sh -O /tmp/cmake.sh
```

Comment:
aarch64?

Reply:
Oops, I used aarch64 cause I'm running docker on a M1 mac for my local tests